Valentina-Atehortua-Zapata/tarea-7/magazine-react/src/components/Footer.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import qr from "../assets/img/barcode1.gif";
+import TitleFooter from "./TitleFooter";
+
+const Footer = () => {
+  return (
+    <div>
```

Comment:
@ValentinaAtehortuaZ you should avoid the div element and use footer as root element